src/v/model/tests/BUILD

```diff
@@ -0,0 +1,91 @@
+load("//bazel:build.bzl", "redpanda_cc_library")
+load("//bazel:test.bzl", "redpanda_cc_btest")
+
+redpanda_cc_library(
+    name = "random",
```

Comment:
Can we make this testonly = True?

Reply:
fixed this, and all the other instances in the tree.